### ISSUE TITLE
How to use Tensorboard Callbacks show the whole process if I fit the model for many times?

### ISSUE THREAD

**Issue by yyh2503** [keras-team/keras#9935]
Hi
I write a loop to train a reinforcement learning model, it is like the followings:

for e in range(episodes):
    model.fit(X1, np_Y, epochs=1)

because the action memory need to be generated every loop, so the "fit" function must be called once every time.

I want to print the loss curve with Tensorboard show the whole process, so I call a keras.callbacks.TensorBoard. And put it in fitting, like:model.fit(X1, np_Y, epochs=1, callbacks=[TensorBoard])
But it generate a new log file every times I call "fit". So it can't draw a curve in Tensorboard when I use tensorboard --logdir=/log_dir.

Please help me how to draw a loss curve show the whole process of all the times of the model fitting?

Thank you!

**Comment by lssily** [comment]
I also got this problem. Have you found any solutions so far？

**Comment by yyh2503** [comment]
@lssily 
I have found it.
You can use train_on_batch instead of fit.
Thanks to Justry for his code:[https://justttry.github.io/justttry.github.io/tensorboard-on-train_on_batch/](url)

**Comment by byamao1** [comment]
I modify the code of [Tensorboard On Train_on_batch](https://justttry.github.io/justttry.github.io/tensorboard-on-train_on_batch/) for epoch train.
[My code](https://www.jianshu.com/p/0ec6462cce6b)
